Topics/Professionalism.md

```diff
@@ -67,6 +67,29 @@ Professionalism is essential in all aspects of software engineering, and maintai
+
+1. **Ask for suggestions or examples of how to solve the issue when unsure of what feedback means or entails.** For example, suppose your boss tells you that you are not engaged enough in online meetings (vague!). Ask your boss for suggestions on how to be more engaged. Your boss might say they want you to talk more (still vague!), but at least you now sense what your boss means. Moving forward, you might want to say “Hello” to everyone when you join the call or give signs of affirmation throughout the meeting, such as nodding your head. You can then follow up with your boss to ask if they feel you have improved. If they say no, then you try something else and ask for more clarification until you’ve narrowed down what exactly was the problem and its corresponding solution.
+2. **Avoid being overly defensive and overwhelmed by taking a step back if the criticism seems too harsh.** You can step back in various ways, but we recommend being honest and communicating that you need a few minutes to compose yourself. This pause will also give the critic time to adjust their tone and thoughts so that the conversation can be more productive and beneficial to everyone involved.
+3. **Acknowledge valid points that the critic gives by stating what you agree with** (e.g. “I agree with you that this line of code here could be more readable”). Be very specific here so that there is no misunderstanding caused later on.
+4. **Listen attentively**, especially when they can see your face and body. What this means is to ensure that your body or, at least, your head is facing them, and ideally, you will give signs that show you are following along, such as nodding your head and so on. However, the more important thing is that you are paying attention to what the critic is saying and thinking about how you will respond using the first three tips to guide what you could think about.
+5. **Do additional research if necessary and fully understand the feedback before replying or making changes**, especially for code reviews. Consult with the reviewer in question and ask for feedback directly, which may be more helpful and faster. Especially for code, make sure to go back and check other code that you wrote to see if you can apply the same feedback and make improvements.
```

Comment:
I would just add that when possible and reasonable, try to understand the feedback and do any necessary additional research **yourself** before asking the reviewer (within reason, if it seems like it's going to take an immense amount of time whereas the reviewer could help you quickly then you should certainly ask right away). This shows that you are willing to put the effort in and trying to learn and improve, as opposed to just resolving feedback as lazily as possible and burdening other people to fix your work.

Reply:
Ah true, I had this in mind when I wrote it, but I don't think I conveyed it as explicitly as I wanted to. Thank you for the suggestion, I've made changes accordingly.